### PR TITLE
Issues/code cleanup py3 compat

### PR DESCRIPTION
**Pull request by mkoistinen**

### CODE REVIEW

File: aldryn_jobs/admin.py [hunk 1]
```diff
@@ -23,9 +27,11 @@
 
 def _send_rejection_email(modeladmin, request, queryset, lang_code='',
                           delete_application=False):
-    # 1. send rejection email - this should be refactored to use djangos "bulk" mail
```
Comment (Chive) [inline aldryn_jobs/admin.py]: I'd rather have docstrings ``"""`` here

File: aldryn_jobs/admin.py [hunk 2]
```diff
@@ -37,10 +43,13 @@ def _send_rejection_email(modeladmin, request, queryset, lang_code='',
     qs_count = queryset.count()
     if not delete_application:
         queryset.update(is_rejected=True, rejection_date=now())
-        success_msg = _("Successfully sent %(count)s rejection email(s).") % {'count': qs_count, }
+        success_msg = _("Successfully sent %(count)s rejection email(s).") % {
```
Comment (Chive) [inline aldryn_jobs/admin.py]: use str.format here

File: aldryn_jobs/admin.py [hunk 3]
```diff
@@ -37,10 +43,13 @@ def _send_rejection_email(modeladmin, request, queryset, lang_code='',
...
+            'count': qs_count,
+        }
     else:
         queryset.delete()
-        success_msg = _("Successfully deleted %(count)s application(s) and sent rejection email.") % { 'count': qs_count, }
+        success_msg = _("Successfully deleted %(count)s application(s) and "
```
Comment (Chive) [inline aldryn_jobs/admin.py]: str.format

File: aldryn_jobs/admin.py [hunk 4]
```diff
@@ -63,7 +72,9 @@ class SendRejectionEmailAndDelete(SendRejectionEmail):
     def __init__(self, lang_code=''):
         super(SendRejectionEmailAndDelete, self).__init__(lang_code)
         self.name = 'send_rejection_and_delete_{0}'.format(self.lang_code)
-        self.title = _("Send rejection e-mail and delete application %s" % self.lang_code)
+        self.title = _("Send rejection e-mail and delete application %s") % (
```
Comment (Chive) [inline aldryn_jobs/admin.py]: str.format .. :)

File: aldryn_jobs/admin.py [hunk 5]
```diff
@@ -107,17 +120,19 @@ def get_actions(self, request):
...
 
     def has_add_permission(self, request):
-        # Don't allow creation of "new" applications via admin-backend until it's properly implemented
+        # Don't allow creation of "new" applications via admin-backend until
+        # it's properly implemented
         return False
 
     def get_attachment_address(self, instance):
-        attachment_link = u'<a href="%(address)s">%(address)s</a>'
+        attachment_link = '<a href="%(address)s">%(address)s</a>'
```
Comment (Chive) [inline aldryn_jobs/admin.py]: str.format

File: aldryn_jobs/managers.py [hunk 6]
```diff
@@ -66,23 +70,28 @@ def send_job_notifiation(self, recipients=None, job_list=None, current_domain=No
         # right now it does not makes a lot of sense but in future someone
         # will just change logic here
         if job_list is None:
-            print "Can't send jobs newsletter without job list to be sent."
+            # TODO: We probably shouldn't just print to console, who would ever
+            # see it?
+            print("Can't send jobs newsletter without job list to be sent.")
```
Comment (Chive) [inline aldryn_jobs/managers.py]: print is not gonna ``message admin``, as is?

File: aldryn_jobs/models.py [hunk 7]
```diff
@@ -194,40 +197,33 @@ def get_absolute_url(self, language=None):
...
 
+
 @version_controlled_content(follow=['category', 'app_config'])
+@python_2_unicode_compatible
 class JobOffer(TranslatableModel):
     translations = TranslatedFields(
         title=models.CharField(_('Title'), max_length=255),
-        slug=models.SlugField(
-            _('Slug'), max_length=255, blank=True,
+        slug=models.SlugField(_('Slug'), max_length=255, blank=True,
```
Comment (Chive) [inline aldryn_jobs/models.py]: what I tend to do with models definitions is:

```python
slug = models.SlugField(
    _('Slug'), max_length=255, blank=True,
    help_text=_('Auto-generated. Used in the URL. If changed, the URL '
                'will change. Clean it to have it re-created.')
)
```

Comment (Chive) [inline aldryn_jobs/models.py]: please put whitespaces between ``slug=models`` :) same above for title

File: aldryn_jobs/models.py [hunk 8]
```diff
@@ -394,7 +391,7 @@ def send_newsletter_confirmation_email(self, request=None):
 
         # get site domain
         full_link = '{0}{1}'.format(get_current_site(request).domain,
-                                          self.get_absolute_url())
+                                    self.get_absolute_url())
```
Comment (Chive) [inline aldryn_jobs/models.py]: how about

```python
full_link = '{0}{1}'.format(
    get_current_site(request).domain,
    self.get_absolute_url()
)
```

File: aldryn_jobs/admin.py (hunk 1 above)
Comment (mkoistinen) [inline aldryn_jobs/admin.py]: This isn't a heredoc, its just a comment.

File: aldryn_jobs/models.py (hunk 7 above)
Comment (mkoistinen) [inline aldryn_jobs/models.py]: For "normal" field definitions, yes, but this is actually a parameter for the translations() method. PEP8 frowns upon spaces here.

File: aldryn_jobs/admin.py (hunk 1 above)
Comment (Chive) [inline aldryn_jobs/admin.py]: ok.

File: aldryn_jobs/models.py (hunk 7 above)
Comment (Chive) [inline aldryn_jobs/models.py]: oh - didn't see. i wondered why

**Comment by Chive** [comment]
Fantastic! :shipit:

**Comment by Chive** [comment]
LGTM

**Pull request merged**